graphics/pympress/Portfile

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        cimbali pympress 1.8.5 v
+revision            0
+
+categories          graphics python
+supported_archs     noarch
+platforms           {darwin any}
+license             GPL-2
+maintainers         {@flyingsamson tuxcad.de:flyingsamson} openmaintainer
+description         PDF presentation tool
+long_description    ${name} is a {*}${description} designed for dual-screen setups such \
```

Comment:
You should probably wrap this at 80 columns.

Reply:
Good point! I rewrapped that line now.